guides/v2.3/config-guide/mq/manage-message-queues.md

```diff
@@ -57,7 +57,7 @@ Edit */app/etc/env.php* file for configure cron job `consumers_runner`
 ...
 ```
 
-* `cron_run` - the option for enabling/disabling cron job `consumers_runner`, by default is true.
+* `cron_run` - the option for enabling/disabling cron job `consumers_runner`, by default is false.
 * `max_messages` - the maximum number of messages for each consumer that must be processed before consumer terminate, by default is 1000. If it is 0, then the consumer never stops working.
 * `consumers` - the list of consumers which will be run, by default is empty array (all consumers are allowed to be run).
```

Comment:
```suggestion
* `consumers` - An array of strings specifying which consumer(s) to run. An empty array runs all consumers.
```

Reply:
@brunoroeder  Thanks for your contribution. I added some suggested updates to clarify the option descriptions, which match the descriptions that are in the _Cloud Guide_ environment variable information related to the `cron_consumer_runner` job.  See [CRON_CONSUMERS_RUNNER](https://devdocs.magento.com/guides/v2.3/cloud/env/variables-deploy.html#cron_consumers_runner).

Reply:
Thanks @meker12

Reply:
@brunoroeder One small suggestion to add code syntax to `consumers_runner` job.  After you accept those changes, we can merge your PR.  Thx!